lib/bbbevents/poll.rb

```diff
@@ -13,6 +13,12 @@ def published?
       @published
     end
 
+    def to_hash
```

Comment:
Please use `to_h` rather than `to_hash` here as well.

Reply:
Renamed to `to_h`